src/components/flow-builder/edge-types/workflow-edge.tsx

```diff
@@ -1,31 +1,42 @@
-import { getBezierPath, type EdgeProps } from "reactflow";
+import { getBezierPath, EdgeLabelRenderer } from "reactflow";
+import Plus from "@pb/components/icons/plus";
+
+import { type CustomEdgeProps } from "../flow-builder.types";
 
 import useEdgeClick from "../hooks/useEdgeClick";
 
+const EdgeButton = ({ onClick }: { onClick: () => void }) => (
+  <button
+    className="edgebutton [&>svg]:stroke-base-100"
+    onClick={onClick}
+    aria-label="Plus button"
```

Comment:
Could you kindly add accessibility attributes like **role**, **tabIndex**, and **type** to further enhance the accessibility of the button?

Reply:
it's not necessary at the moment but i will add it